src/app_service/service/community_tokens/service.nim

```diff
@@ -1411,10 +1411,13 @@ QtObject:
   proc getCommunityTokenOwners*(self: Service, communityId: string, chainId: int, contractAddress: string): seq[CommunityCollectibleOwner] =
     return self.tokenOwnersCache.getOrDefault((chainId: chainId, address: contractAddress))
 
+  proc isTokenDeployed*(self: Service, token: CommunityTokenDto): bool =
+    return token.deployState == DeployState.Deployed
+
   proc onRefreshTransferableTokenOwners*(self:Service) {.slot.} =
     let allTokens = self.getAllCommunityTokens()
     for token in allTokens:
-      if token.transferable:
+      if token.transferable and self.isTokenDeployed(token):
```

Comment:
I think it would be safer to check it in fetchCommunityOwners() function, because it is used in a few places.